docs/content.zh/docs/dev/table/sql/queries/overview.md

```diff
@@ -155,110 +158,116 @@ sink_descriptor = TableDescriptor.for_connector("filesystem")
-* For batch jobs or streaming jobs without checkpointing, `TableResult.collect()` and `TableResult.print()` have neither exactly-once nor at-least-once guarantee. Query results are immediately accessible by the clients once they're produced, but exceptions will be thrown when the job fails and restarts.
-* For streaming jobs with exactly-once checkpointing, `TableResult.collect()` and `TableResult.print()` guarantee an end-to-end exactly-once record delivery. A result will be accessible by clients only after its corresponding checkpoint completes.
-* For streaming jobs with at-least-once checkpointing, `TableResult.collect()` and `TableResult.print()` guarantee an end-to-end at-least-once record delivery. Query results are immediately accessible by the clients once they're produced, but it is possible for the same result to be delivered multiple times.
+*   对于没有checkpointing的批式或流式作业, `TableResult.collect()` 和 `TableResult.print()` 既不保证精确一次(exactly-once)也不保证至少一次(at-least-once) .查询结果一旦产生,客户端可以立即访问,但是,作业失败或重启将抛出异常.
+*   对于checkpointing设置为精确一次(exactly-once)的流式作业, `TableResult.collect()` 和 `TableResult.print()` 保证端到端(end-to-end)的数据只传递一次.相应的checkpoint完成后,客户端才能访问结果.
+*   对于checkpointing设置为至少一次(at-least-once)的流式作业, `TableResult.collect()` 和 `TableResult.print()` 保证端到端(end-to-end)的数据至少传递一次. 查询结果一旦产生,客户端可以立即访问,但是可能会有同一条数据出现多次的情况.
 
 {{< tabs "88a003e1-16ea-43cc-9d42-d43ef1351e53" >}}
 {{< tab "Java" >}}
+
 ```java
 StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
 StreamTableEnvironment tableEnv = StreamTableEnvironment.create(env, settings);
 
 tableEnv.executeSql("CREATE TABLE Orders (`user` BIGINT, product STRING, amount INT) WITH (...)");
 
-// execute SELECT statement
+// 执行`SELECT`语句
 TableResult tableResult1 = tableEnv.executeSql("SELECT * FROM Orders");
-// use try-with-resources statement to make sure the iterator will be closed automatically
+// 使用 try-with-resources 确保iterator会自动关闭
 try (CloseableIterator<Row> it = tableResult1.collect()) {
     while(it.hasNext()) {
         Row row = it.next();
-        // handle row
+        // 处理数据
     }
 }
 
-// execute Table
+// 执行表
 TableResult tableResult2 = tableEnv.sqlQuery("SELECT * FROM Orders").execute();
 tableResult2.print();
 
 ```
+
 {{< /tab >}}
 {{< tab "Scala" >}}
+
 ```scala
 val env = StreamExecutionEnvironment.getExecutionEnvironment()
 val tableEnv = StreamTableEnvironment.create(env, settings)
-// enable checkpointing
+// 启用 checkpointing
 tableEnv.getConfig
   .set(ExecutionCheckpointingOptions.CHECKPOINTING_MODE, CheckpointingMode.EXACTLY_ONCE)
 tableEnv.getConfig
   .set(ExecutionCheckpointingOptions.CHECKPOINTING_INTERVAL, Duration.ofSeconds(10))
 
 tableEnv.executeSql("CREATE TABLE Orders (`user` BIGINT, product STRING, amount INT) WITH (...)")
 
-// execute SELECT statement
+// 执行`SELECT`语句
```

Comment:
It is not recommended to translate the code comment.

Reply:
already modified.